src/org/usfirst/frc/team1923/robot/commands/drive/DriveDistanceCommand.java

```diff
@@ -78,33 +80,43 @@ protected void execute() {
         Robot.driveSubSys.drive(this.leftTarget, this.rightTarget, TalonControlMode.Position);
     }
 
+    /**
+     * Finished if - timed out or - left and right errors are greater then the
+     * allowable error - and current positions do not equal initial positions
+     */
     @Override
     protected boolean isFinished() {
-        if (RobotMap.DEBUG) {
+        if (RobotMap.DEBUG)
             System.out.println("Left Error: " + Robot.driveSubSys.getLeftError() + ", Right Error: " + Robot.driveSubSys.getRightError());
-        }
-        return isTimedOut() || ((Math.abs(Robot.driveSubSys.getLeftError()) < Robot.driveSubSys.ALLOWABLE_ERROR)
-                && (Math.abs(Robot.driveSubSys.getRightError()) < Robot.driveSubSys.ALLOWABLE_ERROR)
-                && Robot.driveSubSys.getLeftPosition() != this.oldleft && Robot.driveSubSys.getRightPosition() != this.oldright);
+
+        return isTimedOut() || Math.abs(Robot.driveSubSys.getLeftError()) < Robot.driveSubSys.ALLOWABLE_ERROR

```

Comment:
Make sure that the minimalistic grouping is easy to understand bc others might now know the precedences